marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/graph/searchNodes.sjs

```diff
@@ -0,0 +1,203 @@
+/**
+ Copyright (c) 2021 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+'use strict';
+
```

Comment:
Need to add a comment that starts with  `// No privilege required:` to explain that no special privilege is needed for this endpoint.

Reply:
I added one comment,  just tell me if another message could explain it better. thank you!